Parsers/ASimDns/Parsers/ASimDnsMicrosoftNXlog.yaml

```diff
@@ -1,17 +1,19 @@
 Parser:
   Title: DNS activity ASIM parser for Microsoft DNS logs collected using NXlog
-  Version: '0.4'
-  LastUpdated: Dec 11 2022
+  Version: '0.5'
```

Comment:
@jszigetvari-nxlog Also, noticed the version should be 0.5.0

Reply:
corrected and pushed.